environment/lib/config.js

```diff
@@ -132,7 +132,11 @@ class RainbowConfig {
 // value is coming from.
 RainbowConfig.paramValues = {};
 
-RainbowConfig.optionalValues = ['bridge-src', 'core-src', 'nearup-src'];
+RainbowConfig.optionalValues = {
```

Comment:
It is not clear what was the benefit of introducing `RainbowConfig.optionalValues` initially. Also, this is not the right place for it, if we have an optional value with default param here is how it can be specified: https://github.com/near/rainbow-bridge/blob/198769b68feac87eb8d3a258441c4dc663b3ed2f/environment/index.js#L83

Also, let's stop using empty string `''` as the default value for our arguments.